src/operator/batch_norm_v1-inl.h

```diff
@@ -261,7 +261,7 @@ class BatchNormV1Prop : public OperatorProperty {
     using namespace mshadow;
     CHECK_EQ(in_shape->size(), 3U) << "Input:[data, gamma, beta]";
     const mxnet::TShape &dshape = in_shape->at(0);
-    if (!shape_is_known(dshape)) return false;
+    if (dshape.ndim() == -1) return false;
```

Comment:
i don't understand this modification. ndim == -1 means unknown shape for numpy convention. ndim == 0 means unknown shape for mxnet convention. Why do we only need to check for numpy convention?